salt/postgres/backup.sls

```diff
@@ -10,16 +14,27 @@ pgbackrest:
 
 /etc/pgbackrest/pgbackrest.conf:
   file.managed:
-   - makedirs: True
-   - user: postgres
-   - group: postgres
-   - source: salt://postgres/files/pgbackrest/{{ pillar.postgres.backup.configuration }}.conf
-   - template: jinja
+    - makedirs: True
+    - user: postgres
+    - group: postgres
+    - source: salt://postgres/files/pgbackrest/{{ pillar.postgres.backup.configuration }}.conf
+    - template: jinja
+
+{% elif salt['pillar.get']('postgres:backup:type') == 'script' %}
+{{ set_config('aws-settings.local', 'S3_DATABASE_BACKUP_BUCKET', pillar.postgres.backup.location ) }}
```

Comment:
This is the same variable used for MySQL backups. If we have a server with both MySQL and PostgreSQL, will the backups conflict, or will they be fine to share the bucket?

Reply:
If we had both backup scripts on one server the backups would be muddled together and salt would clash setting this variable twice. 
Backup file names would be especially confusing if the databases had similar names between services, in this case, the only difference would be the file name ending either `.sql.gz` (MySQL) or `.tar.gz` (PostgreSQL).

However, it is very unlikely and it would be bad design to have MySQL and PostgreSQL on the same server since they both fill the same task in the stack - so this shouldn't be a problem.